typing/typecore.ml

```diff
@@ -415,12 +415,12 @@ let unify_pat_types_return_equated_pairs ~refine loc penv ty ty' =
 let unify_pat_types_refine ~refine loc penv ty ty' =
   ignore (unify_pat_types_return_equated_pairs ~refine loc penv ty ty')
 
-(** [sdesc_for_hint] is used by error messages to report literals in their
+(** [sexp_for_hint] is used by error messages to report literals in their
     original formatting *)
-let unify_pat ?sdesc_for_hint env pat expected_ty =
+let unify_pat ?sexp_for_hint env pat expected_ty =
```

Comment:
This PR is still only using the description for hints in the pattern case. I propose to keep the code unchanged.

Reply:
Done.